lib/server.js

```diff
@@ -40,8 +40,8 @@ function IMAPServer(options) {
 
     this.options = options || {};
     this.options.credentials = this.options.credentials || {
-        key: fs.readFileSync(__dirname + "/../cert/server.key"),
-        cert: fs.readFileSync(__dirname + "/../cert/server.crt")
+        key: fs.readFileSync(__dirname + "/../../../certs/server.key"),
+        cert: fs.readFileSync(__dirname + "/../../../certs/server.crt")
```

Comment:
Why change the certs dir?

Reply:
ignore this. sorry.  I will revert..sorry.....